contracts/contracts/MACI.sol

```diff
@@ -99,17 +98,17 @@ contract MACI is IMACI, DomainObjs, Params, SnarkCommon, Ownable {
     event MergeStateAqSubRoots(uint256 _pollId, uint256 _numSrQueueOps);
     event MergeStateAq(uint256 _pollId);
 
-    /*
-    * Ensure certain functions only run after the contract has been initialized
-    */
+    /**
+     * @notice Ensure certain functions only run after the contract has been initialized
```

Comment:
Prob makes sense to use a consistent commenting format? Could this be added to the linter @0xmad?

Reply:
let's go with /// for both single and multiline comments? https://docs.soliditylang.org/en/v0.8.20/natspec-format.html in their examples they use ///

Reply:
I'm down. Seems like it'd make it easier for changing a comment from single to multi-line? Or avoids the need for 3 lines when making a single-line comment.